uvicorn/protocols/websockets/websockets_impl.py

```diff
@@ -50,7 +50,6 @@ def __init__(self, config, server_state, _loop=None):
         self.initial_response = None
         self.connect_sent = False
         self.accepted_subprotocol = None
-        self.transfer_data_task = None
```

Comment:
Based on my other comment, it's likely this `transfer_data_task` attribute is unrelated to resolving this bug (or at least, there are other ways to tell if the handshake has failed). So not sure if we should just keep this line for the sake of reducing the diff footprint, or go ahead and remove this since we're obviously not using it anywhere else.